src/cli/command.rs

```diff
@@ -42,6 +42,10 @@ pub enum Command {
     /// Display schema
     #[arg(short, long)]
     schema: bool,
+
+    /// Operations to check
+    #[arg(short, long, num_args = 1..)]
+    operations: Vec<String>,
```

Comment:
Add [value_delimiter](https://docs.rs/clap/latest/clap/struct.Arg.html#method.value_delimiter) this to the arg so that even `,` comma separated values get captured.

Reply:
Thank you.